ArjunaJTS/interop/pom.xml

```diff
@@ -51,6 +52,43 @@
     </repository>
   </repositories>
 
+  <dependencies>
+    <dependency>
+      <groupId>org.jboss.spec</groupId>
+      <artifactId>jboss-javaee-7.0</artifactId>
+      <version>1.1.1.Final</version>
```

Comment:
please, could you move it to the property instead of the direct definition here?

It will be later easier to move the definition to the parent `pom.xml` which would be desirable especially for this ee spec definitions - as they should be used in the same version over the all quickstarts (if possible).
(^^^ I don't mean this move to parent pom should be done as part of this PR. I'm just argumenting why it's desirable).

Reply:
sure